packages/v4/patternfly-docs/content/design-guidelines/components/empty-state/empty-state.md

```diff
@@ -35,7 +29,14 @@ There are several common uses for empty states:
 - [Success](#success)
 - [Creation](#creation) 
 
-### Getting started 
+## Variations
```

Comment:
Would re-organize this so that all the "Use cases" are BEFORE the variations since the use cases are more reflective of usage than variations. Right now it's calling out usage, then variations, just to go back to use cases haah

Reply:
Okay so for this one, Allison and I were kind of going back and forth because we felt like it was weird to mention all the different use cases before the variations are stated. Since each particular use case goes along with a certain variation. So instead we chose to do the one-liner under usage and then hyperlinked to the sections where it discusses usage with the proper variation. However, I am realizing the hyperlinks were off so im changing those...

We also did it like that on a few other components where the elements were different based on variations. So under the elements, we stated "elements vary depending on component variation" and then hyperlinked. We personally felt like that was the best way because we want to keep the ordering of headers consistent..

What are your thoughts on that? This one is a little weirder because there are a bunch of use types and then alsooo variations so what if I made the one-liner "There are several common use cases for empty states that correspond  with different variations, see variations section for explanations." Orrrr "Use cases correspond with different component variations"  something like that? 

Orrr I can list all the use cases and then under variations just have a list of the different use cases and what variation they should use..

Sorry that was a lot hahaha let me know what you think or if that didn't make any sense!

Reply:
Ok I think that organization is great with the "use a X variation for this one" line under each of them!! The only thing I am iffy about is the bulleted H3s hahaha like it just looks so weird to have headers in bullets... so I think maybe don't bullet it, just list it out with one liners under each and that's fine - if we wanna go the extra mile we'd add an image under each one, but then it's kinda repetitive since we already had images... so maybe you can link to the use case examples in the variations as well. So for the extra small, link to the card example